components/eamxx/cime_config/namelist_defaults_eamxx.xml

```diff
@@ -187,6 +187,10 @@ be lost if SCREAM_HACK_XML is not enabled.
       <!-- Frequency in physics steps to output a global hash over the dycore's
            in-fields. <= 0 disables hashing. -->
       <BfbHash type="integer">18</BfbHash>
+      <enable_column_conservation_checks>false</enable_column_conservation_checks>
+      <enable_energy_fixer>true</enable_energy_fixer>
+      <!-- true means 2 more calls to AllReduce after dycore -->
+      <enable_energy_fixer_debug_info>true</enable_energy_fixer_debug_info>
```

Comment:
I think the extra debug output shouldn't default to `true`.

Reply:
thanks, i do not want functionality of ${DEBUG} there, switching to false.